brainglobe_atlasapi/atlas_generation/volume_utils.py

```diff
@@ -14,7 +14,10 @@
 
 import os
 
+######## I had to manually install this
```

Comment:
We should depend on `brainglobe-utils.imageio` instead (see https://github.com/brainglobe/brainglobe-utils/pull/33 for details), which may need to be added to the dependencies.